src/test/java/com/wikia/webdriver/testcases/desktop/auth/SignupTests.java

```diff
@@ -64,38 +63,62 @@ public void userCanSignUpWithExistingEmailMobile() {
     performSignUpOnMobileAs(createUserWithExistingEmail());
   }
 
+  @DontRun(language = "szl")
   @Test(groups = DESKTOP)
   public void userCannotSignUpWithExistingUsernameDesktop() {
     RegisterPage form = performSignUpExpectingFailureOnDesktopAs(createUserWithExistingUsername());
     assertEquals(form.getError(), USERNAME_TAKEN_MSG);
   }
 
+  @RunOnly(language = "szl")
+  @Test(groups = DESKTOP)
+  public void userCannotSignUpWithExistingUsernameDesktopSzl() {
+    RegisterPage form = performSignUpExpectingFailureOnDesktopAs(createUserWithExistingUsername());
+    assertEquals(form.getError(), "Nazwa użytkownika jest już wykorzystywana");
```

Comment:
Would be cool to turn these strings into const values and use it, such as is done with i.e. `PASSWORD_MATCHING_USERNAME_MSG`. But it's not a game changer of course. :)